package.json

```diff
@@ -1,31 +1,51 @@
 {
   "name": "powerbi-visuals-sunburst",
   "description": "SunBurst chart.",
-  "version": "1.0.5",
+  "version": "1.0.6",
   "author": {
     "name": "Microsoft",
     "email": "pbicvsupport@microsoft.com"
   },
   "scripts": {
-    "postinstall": "typings install && pbiviz update 1.3.0",
-    "typings": "typings",
+    "postinstall": "pbiviz update 1.5.0",
     "pbiviz": "pbiviz",
     "start": "pbiviz start",
     "package": "pbiviz package",
-    "lint": "node node_modules/tslint/bin/tslint \"+(src|test)/**/*.ts\""
+    "lint": "tslint \"+(src|test)/**/*.ts\"",
```

Comment:
please modify to:
 "lint": "node node_modules/tslint/bin/tslint -r \"node_modules/tslint-microsoft-contrib\"  \"+(src|test)/**/*.ts\"",